modules/seqexec/model/shared/src/main/scala/seqexec/model/UserPrompt.scala

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model
+
+import cats.Eq
+import cats.implicits._
+import gem.Observation
+
+sealed trait UserPrompt extends Product with Serializable
+
+object UserPrompt {
```

Comment:
Upon second look, maybe we could keep the UI details on the client side? As per our Slack conversation, I understand that adhering to this would mean changing `Notification` too.